azure/eventhub_log_forwarder/activity_logs_deploy.ps1

```diff
@@ -25,14 +28,25 @@ New-AzResourceGroupDeployment `
     -functionAppName $FunctionAppName `
     -functionName $FunctionName `
     -datadogSite $DatadogSite `
-    -Verbose
+    -Verbose `
+    -ErrorAction Stop
```

Comment:
Does this still prevent meaningful error messages from being written like we'd seen in end to end testing?

Reply:
I was able to still see error messages logged when I added it back in, and I tested in cloudshell as well - but since you're more familiar with the cloudshell piece maybe you can test to make sure it looks good to you?